src/EntityGraphQL/Compiler/EntityGraphQLQueryWalker.cs

```diff
@@ -373,7 +373,7 @@ private Dictionary<string, object> ProcessArguments(IField field, IEnumerable<Ar
                 throw new EntityGraphQLCompilerException($"No argument '{argName}' found on field '{field.Name}'");
             }
             var r = ParseArgument(argName, field, arg);
-            if (r != null)
+            //if (r != null)
```

Comment:
Also clean this up. Assuming we always add now as we handle `null` as a missing (not provided) argument